pkg/cloud/services/asg/autoscalinggroup.go

```diff
@@ -45,6 +47,15 @@ func (s *Service) SDKToAutoScalingGroup(v *autoscaling.Group) (*expinfrav1.AutoS
 		i.Tags = converters.ASGTagsToMap(v.Tags)
 	}
 
+	if len(v.Instances) > 0 {
+		for _, autoscalingInstance := range v.Instances {
+			tmp := &infrav1.Instance{
+				ID: aws.StringValue(autoscalingInstance.InstanceId),
+			}
+			i.Instances = append(i.Instances, *tmp)
```

Comment:
I was going to suggest renaming `tmp` to something different, but when I really thought about it, I couldn't think of a good suggestion. 
`instance` came to mind, but that might get confusing with `infrav1.Instance` and `i.Instances`

It's really up to you 😄 I think it might be fine as is 😂

Reply:
yeah. i wonder if i can put everything into line 55. i used `tmp` in multiple places so i'll open a new PR to clean that stuff up if that's okay